libmscore/rest.cpp

```diff
@@ -108,8 +118,14 @@ void Rest::draw(QPainter* painter) const
             painter->drawLine(QLineF(0.0, -_spatium, 0.0, _spatium));
             painter->drawLine(QLineF(x2,  -_spatium, x2,  _spatium));
 
+            // draw number
             std::vector<SymId>&& s = toTimeSigString(QString("%1").arg(n));
-            qreal y = -_spatium * 1.5 - staff()->height() *.5;
+
+//            if (isStyled(Pid::YPOS_MMREST_NUMBER))
+//                  _yPosMMRestNumber = propertyDefault(Pid::YPOS_MMREST_NUMBER).toPointF();
+
+            qreal y = _yPosMMRestNumber * spatium() - staff()->height() *.5;
+            // TODO: IF within staff, mask (paint out) a slightly larger rectangle so number shows clearly
```

Comment:
It might be easier to avoid drawing the line (done at line 113) instead of painting a rectangle above, simply
1) painting a white rectangle will probably hide staff lines
2) it will avoid hiding anything else under the rectangle
3) deciding which color to draw the rectangle might not be as easy as it seems. On some coloured staves, white might not be the best option.

Reply:
Good idea!

Reply:
How would you go about determining the width of the number drawn?

Reply:
Good question...
I've seen some code related to that in the code base. I'll let you know when I find it.

Reply:
@Marr11317 any ideas on that measurement? I'm not against leaving that TODO in the code and calling this a useful feature already, but I'll have time in the next day or two to return to this.

Reply:
Added in latest push